projects/canviewer/canserver.py

```diff
@@ -0,0 +1,40 @@
+import cantools
```

Comment:
Add a comment about the purpose of this file. Why are we creating a thread? What is it doing?

Reply:
Echoing this! From the perspective of someone who doesn't really have a good background on the way the system works, it'd be really nice to have explanations for individual scripts as well as the readme for the whole system.